db/fixtures/ae_datastore/ManageIQ/Cloud/VM/Provisioning/Placement.class/__methods__/best_fit_google.rb

```diff
@@ -0,0 +1,26 @@
+###################################################################
+#
+# Description: Select the cloud network and availability zone for Google
+#
+###################################################################
+
+prov  = $evm.root["miq_provision"]
+image = prov.vm_template
+raise "Image not specified" if image.nil?
+
+if prov.get_option(:availability_zone).nil?
+  availability_zone = prov.eligible_availability_zones.first
+
+  if availability_zone
+    prov.set_availability_zone(availability_zone)
```

Comment:
@billfitzgerald0120 I would expect to see an "info" logging message for AZ like we have for the `cloud_network` below.